website/common/script/content/constants/events.js

```diff
@@ -46,13 +46,13 @@ export const REPEATING_EVENTS = {
     ],
   },
   piDay: {
-    start: new Date('1970-03-13T08:00-05:00'),
-    end: new Date('1970-03-15T23:59-05:00'),
+    start: new Date('1970-03-13T04:00-05:00'),
```

Comment:
Start date is march 14th. Timezone offset is `-04:00`

Reply:
Daylight Savigs Time is the worst.